src/hooks/useManualPatronBlocks/useManualPatronBlocks.test.js

```diff
@@ -0,0 +1,45 @@
+import { act } from 'react';
+
+import {
+  QueryClient,
+  QueryClientProvider,
+} from 'react-query';
+
+import { renderHook } from '@folio/jest-config-stripes/testing-library/react';
+import { useOkapiKy } from '@folio/stripes/core';
+
+import useManualPatronBlocks from './useManualPatronBlocks';
+
+const queryClient = new QueryClient();
+
+const wrapper = ({ children }) => (
+  <QueryClientProvider client={queryClient}>
+    {children}
+  </QueryClientProvider>
+);
+
+const data = { manualblocks: [{ type: 'Manual' }] };
+
+describe('useManualPatronBlocks', () => {
+  const mockGet = jest.fn(() => ({
+    json: () => Promise.resolve(data),
+  }));
+
+  const kyMock = {
+    get: mockGet,
+  };
+
+  beforeEach(() => {
+    mockGet.mockClear();
+    useOkapiKy.mockReturnValue(kyMock);
+  });
+
+  it('should fetch patron group by id', async () => {
+    const { result } = renderHook(() => useManualPatronBlocks({ userId: 'id' }), { wrapper });
+
+    await act(() => !result.current.isLoadingManualPatronBlocks);
+
+    expect(mockGet).toHaveBeenCalledWith('manualblocks?query=(userId==id)&limit=10000');
```

Comment:
I suggest create variable and use for limit when necessary across all module
`const MAX_RECORDS = '10000';`

Reply:
This limit is for this request only. As for tests, I prefer not to use variables, because if a variable is unintentionally changed, the test will not catch the error.